src/styles/components/header/header.module.scss

```diff
@@ -0,0 +1,81 @@
+@import url('https://fonts.googleapis.com/css?family=Jost:wght@200..900');
+@import url('https://fonts.googleapis.com/css?family=Be+Vietnam:wght@200..900');
+
+.header {
+    display: flex;
+    flex-direction: column;
+    background-color: var(--maroon, #5B1110);
+    height: 226px;
+    justify-content: space-around;
+    
+}
+
+.title {
+    display: flex;
+    flex-direction: row;
+    column-gap: 11px;
+    margin-top: 0.5rem;
+
+    > *:first-child {
+        margin-left: 34px; 
+    }
+}
+
+.title_text {
+    color: #FFFBF0;
+    h1 {
+        font-family: "Malayalam MN";
+        font-size: 2.8125rem;
+        font-style: normal;
+        font-weight: 400;
+        line-height: normal;
+    }
+
+    h2 {
+        font-family: "Be Vietnam";
+        font-size: 1.875rem;
+        font-style: normal;
+        font-weight: 500;
+        line-height: normal;
+    }
+}
+
+.pages {
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    justify-content: space-evenly;
+    align-items: center;
+    
+    a {
+        color: var(--Gold, #D79C1A);
+        text-align: center;
+        font-family: Jost;
+        font-size: 1.5rem;
+        font-style: normal;
+        font-weight: 600;
+        line-height: normal;
+        text-decoration: none;
+        transition: border-bottom 0.3s ease;
+        padding: 10px 30px;
+
+        &:hover {
+            border-bottom: 2px solid var(--Gold, #D79C1A);
+        }
+
+    }
+
+   
+    
+}
+
+
+.page_link.active {
```

Comment:
It kinda disappeared from the figma but one of the images on the week 1 assignment sign up sheet shows how the highlight is supposed to look. It should be completely filled gold highlight with maroon text.